src/app/static/tests/unit/components/utils/formatter.test.ts

```diff
@@ -0,0 +1,23 @@
+import { numericFormatter } from "@/utils/formatter";
```

Comment:
We try to make the tests folder structure mirror that of the source, so that tests are easy to find, so I think this shouldn't be under `/tests/components` - just needs to move to `/tests/utils`

Reply:
done